src/components/item/publish/ItemPublishConfiguration.js

```diff
@@ -250,6 +288,10 @@ const ItemPublishConfiguration = ({
         )}
       </Typography>
       <Button
+        disabled={
+          isDisabled ||
+          itemValidationStatus !== ITEM_VALIDATION_STATUSES.SUCCESS
+        }
         variant="outlined"
         onClick={publishItem}
         color="primary"
```

Comment:
If you factor out this button in a new component, you will be able to split the logic of tags / validation.